### ISSUE TITLE
Combined diagram does not show all persons when siblings are selected

### ISSUE THREAD

**Issue by hartenthaler** [Neriderc/GVExport#198]
As far as I understand the option "siblings": shows all siblings of all persons in the diagram, but not the siblings of the persons in the top generation (because their families are in the generation above the top generation).

This works perfectly in the simple mode, but not in the combined mode. In my example
- simple mode: toast message says 30 individuals and 8 families, and they are all shown
- combined mode: toast message says 30 individuals and 10 families, but the diagram shows only 17 individuals and 7 families.

Simple mode shows all the persons and families (blurred for data security):
![Screenshot (248)](https://user-images.githubusercontent.com/6324175/189690841-85c3115e-82f9-41b0-b280-6a6f9b4f590e.png)

Combined mode shows the brother of the proband (proband is a foster child); does not show the siblings of the mother; does show only one sibling of the grandmother (why this one???), no siblings of the grandfather:
![Screenshot (249)](https://user-images.githubusercontent.com/6324175/189690869-d31ed584-a907-49c0-872d-8069d8d47f5c.png)

**Comment by hartenthaler** [comment]
Ah, the one sibling in generations 1 and 2 that is shown, is the only one that was not married. All the married siblings were removed. But the sibling of the starting person is also shown and is not married too. 

I think that in earlier versions of GVExport, in that case, there were dummy families generated in the combined mode.

**Comment by Neriderc** [comment]
> As far as I understand the option "siblings": shows all siblings of all persons in the diagram, but not the siblings of the persons in the top generation (because their families are in the generation above the top generation).

This is how I understand it too.

It was caused by an earlier fix where a record was half complete when using a stop individual. In this case the missing spouses (with spouses disabled) caused the records to be removed. This should not happen so I have made a small change to resolve it which has now been pushed to the main branch. Let me know if you find it doesn't resolve it.
